java/src/jmri/jmrix/dcc4pc/serialdriver/configurexml/ConnectionConfigXml.java

```diff
@@ -26,7 +26,7 @@ public ConnectionConfigXml() {
     @Override
     @SuppressWarnings("deprecation") // until DCC4PC is migrated to multiple systems
     protected void getInstance() {
-        adapter = SerialDriverAdapter.instance();
+        adapter = new SerialDriverAdapter();
```

Comment:
Not sure if it matters, but `SerialDriverAdapter.instance()` did:
```
            SerialDriverAdapter m = new SerialDriverAdapter();
            m.setManufacturer(Dcc4PcConnectionTypeList.DCC4PC);
            mInstance = m;
```
But the code here omits the line:
`m.setManufacturer(Dcc4PcConnectionTypeList.DCC4PC);`

Reply:
Thanks for checking.  Thats done in jmri/jmrix/dcc4pc/serialdriver/SerialDriverAdapter for multiple-system setups